README

```diff
@@ -25,5 +26,31 @@
  - make install
 
  Make sure that you do not have libcoap-1-0-dev installed, as the headers might conflict.
+
 ===
+
+ Install TPM2 Simulator:
+ - Download code from https://sourceforge.net/projects/ibmswtpm2/
+ - Unpack into any directory
+ - cd ./src
+ - make
+ - rm -f NVChip; ./tpm_server
+
+ Install tpm2-tools:
+ - sudo apt install tpm2-tools or
+ - git clone https://github.com/tpm2-software/tpm2-tools.git
+ - cd tpm2-tools
+ - ./bootstrap
+ - ./configure --prefix=/usr
+ - make -j
+ In case if you build tpm2-tools from source check up the list of dependencies:
+ https://tpm2-tools.readthedocs.io/en/latest/INSTALL/
```

Comment:
Can we containerize the development environment? This one does too portable or easy to setup. Most (or perhaps all) of the stuff we need is already done in the charra project: https://github.com/Fraunhofer-SIT/charra/blob/master/Dockerfile#L64

Reply:
sounds good. I have to do so